### DISCUSSION TITLE
Clearly a need for a secure way to expose Immich publicly

### DISCUSSION THREAD

**Discussion by gene1wood** [immich-app/immich#14288] [General]
I keep seeing these projects pop up :

* Immich Share Proxy : https://github.com/11notes/docker-immich-share-proxy
* Proxy for Immich : https://github.com/tomleb/proxy-for-immich/
* Immich Public Proxy : https://github.com/alangrainger/immich-public-proxy

What feature set is Immich missing that is inspiring these folks to create these projects? Is it already in the works or on an Immich roadmap to introduce this functionality into Immich natively?

From looking at these projects, it sounds like folks aren't comfortable making Immich publicly accessible, but I'm unsure why.

**Comment by mmomjian** [comment]
There is no reason why you can’t publicly expose your albums with a reverse proxy, like many of us do. These users probably want some kind of additional security / something that doesn’t depend on Immich being bug free. It wouldn’t be possible for us to solve this issue because intrinsically it would be an Immich integrated feature (like the current album shares) and thus have security risks in some users eyes.
